### ISSUE TITLE
"clear the queue" after break away

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#103]
From meetings today. @terracoda's testing has found that often there are confusing "warming" alerts that often come through after the break away alert when things should be saying "very hot"

This may be solved by #101, where we will cut out "warmer" and "hotter" alerts from being added after the thermometer is maxed out. If not, and we are finding that these alerts have stacked up in the queue, then we may need to actually flush the `utteranceQueue`. Hence why the title of this issue is in quotes; we don't know if that will be the implementation we go with, or just the desired outcome. 

Tagging @terracoda for comment.

**Comment by terracoda** [comment]
Sounds good, and maybe my comment in #101 (https://github.com/phetsims/friction/issues/101#issuecomment-418823674) should have gone here?

**Comment by terracoda** [comment]
I am not hearing un-timley "warmer" alerts any more, so I think #101 has solved this issue.
Closing.
